webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import TypeAheadSelect from '.';
+import {
+  props,
+  propsWithNew,
+  propsWithPlaceholder,
+  propsWithMultiple,
+  propsWithClearButton,
+} from './TypeAheadSelect.fixtures';
+import Story from '../../../../../..//stories/components/Story';
+import storeDecorator from '../../../../../..//stories/storeDecorator';
+
+export default {
+  title: 'Components|TypeAheadSelect',
+  decorators: [storeDecorator],
+};
+
+export const defaultTypeAhead = () => (
+  <Story>
+    <TypeAheadSelect {...props} />
```

Comment:
The story doesn't look that useful when clicking on **"show code"**, would it make more sense to use hardcoded props instead fixtures?

I'm also asking the same question here:
https://github.com/theforeman/foreman/pull/7355